public/views/create.html

```diff
@@ -61,6 +61,7 @@
             </label>
             <label for="ext-master" class="m10t">
               <span translate>Master extended private key</span>
+              <small translate>If not given, a random key will be generated</small>
```

Comment:
I could not find this string in the template.po file.

I am assuming this is because this fix and the templates.po fix were done on separate branches based off a commit without these changes.